src/EIPTestsFiller/StateTests/stEIP5656-MCOPY/MCOPY_copy_costFiller.yml

```diff
@@ -252,6 +252,7 @@ MCOPY_copy_cost:
           - :label src32_size31
           - :label src32_size32
           - :label src32_size33
+          - :label src0_size44769
```

Comment:
This is because the yul code is now compiled with `PUSH0` (1 gas cheaper than `PUSH1`). I will add the destroyed test case back.

Reply:
Then push0 must not be compiled for legacy networks also